rsciio/delmic/_api.py

```diff
@@ -0,0 +1,79 @@
+import h5py
+import numpy as np
+
+from rsciio._docstrings import FILENAME_DOC, RETURNS_DOC
+
+
+def file_reader(filename, *args, **kwds):

```

Comment:
```suggestion
def file_reader(filename):

```

At the moment `*args` and `**kwds` are not used and just lead to a warning in building the documentation. If added again in the future, they should be documented in the docstring.